grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractServerCall.java

```diff
@@ -266,7 +266,8 @@ final void closeListener(Status newStatus, boolean completed, boolean setRespons
                 ctx.logBuilder().requestContent(GrpcLogUtil.rpcRequest(method, simpleMethodName), null);
             }
             if (setResponseContent) {
-                ctx.logBuilder().responseContent(GrpcLogUtil.rpcResponse(newStatus, firstResponse()), null);
+                ctx.logBuilder().responseContent(GrpcLogUtil.rpcResponse(newStatus, firstResponse(), null),
+                                                 null);
```

Comment:
Should we add `Metadata` as a new method parameter and pass it to `GrpcLogUtil.rpcResponse()` ?

Reply:
Added in https://github.com/line/armeria/pull/4636/commits/264275f48086f9f15ed5ac290f3c183d490dd33d.
But it looks not used in usual, because `setResponseContent` is often `false` for service logic exception in server and client doesn't send error detail when closing the connection.